src/main_thread/decrypt/content_decryptor.ts

```diff
@@ -330,6 +399,14 @@ export default class ContentDecryptor extends EventEmitter<IContentDecryptorEven
     this.trigger("stateChange", this._stateData.state);
   }
 
+  /**
+   * Returns the list of supported codecs with the keysystem configuration.
+   * @returns {Array}: The list of supported codecs.
+   */
+  public getSupportedCodecs(): ICodecSupportList {
+    return this._supportedCodecWhenEncrypted;
+  }
```

Comment:
Why not having something like `isCodecSupported(codec: string): boolean;`?

It would only put this codec specificity (about only testing a few codecs etc.) inside the `decrypt` directory